samples/bufferedReading.ts

```diff
@@ -0,0 +1,26 @@
+import { ProximaStreamClient, Offset, BufferedStreamReader } from "..";
+import { strict as assert } from "assert";
+
+export function sleep(ms: number): Promise<void> {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function main() {
+    const client = new ProximaStreamClient();
+    const name = "proxima.eth-main.blocks.1_0";
+    const pausable = await client.streamEvents(name, Offset.zero);
+    const bufferedStream = BufferedStreamReader.fromStream(pausable);
```

Comment:
let's call it `streamReader` since it's technically not a stream

Reply:
let's pass some `bufferSize` to `.fromStream` method for better illustration of its behavior